en/SUMMARY.md

```diff
@@ -612,6 +613,9 @@
       - [RoverDifferentialGuidanceStatus](msg_docs/RoverDifferentialGuidanceStatus.md)
       - [RoverDifferentialSetpoint](msg_docs/RoverDifferentialSetpoint.md)
       - [RoverDifferentialStatus](msg_docs/RoverDifferentialStatus.md)
+      - [RoverMecanumGuidanceStatus](msg_docs/RoverMecanumGuidanceStatus.md)
+      - [RoverMecanumSetpoint](msg_docs/RoverMecanumSetpoint.md)
+      - [RoverMecanumStatus](msg_docs/RoverMecanumStatus.md)
       - [Rpm](msg_docs/Rpm.md)
```

Comment:
@chfriedrich98 Changes to this file reflect new topics added to PX4 and auto generated in the guide (but not auto added to sidebar).

I am assuming these do not reflect "new documentation that should be written"

Reply:
Not sure if i understand correctly, but these messages are part of the newly added module for mecanum rovers (3rd rover type). This module will get a documention in line with ackermann and differential rovers.

Reply:
@chfriedrich98 No worries - that's what I needed to know.

Let me reword just so it isn't so confusing for you. When these messages were added to the PX4 source the docs were auto updated with new matching topics, but those topics do not automatically go into the sidebar (hence me adding).

That's a hint to me that there is something "new" that might require docs. So my note was a reminder to you that there might be something new that you have forgotten to do. From your note I know that there  is something to document that is on your list :-)